SEO changes to the teachers pension page (#4436)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme.md

---
One of the great benefits of a teaching career is a secure pension that will help you save for your future. 

> The teachers’ pension scheme is one of the most generous in the country.

Author: ```suggestion
```

## How the teachers' pension works

> You’ll automatically be enrolled into the pension scheme at the start of your career. You'll remain in the scheme when moving teaching jobs, unless you opt out. 

Reviewer: ```suggestion
You’ll automatically be enrolled into the pension scheme at the start of your career. You'll remain in the scheme if you move teaching jobs, unless you opt out. 
```
Does everyone always move jobs?

Under the scheme, you'll automatically pay from your salary into your pension each month. At the same time, your employer also contributes.

Once you join the scheme, it's quick and easy to manage your pension online.

## Salary-based pension contributions

The teachers' pension scheme includes an employer contribution of 28.6%.

> It’s a ‘defined benefit’ pension, which means it’s based on your salary rather than the amount you contribute.

Reviewer 2: Do we need the quote marks (I know they are in prod already!)? Pet peeve!
```suggestion
It’s a defined benefit pension, which means it’s based on your salary rather than the amount you contribute.
```

> If your salary increases, you and your employer's pension contributions will increase too. You can also choose to pay extra contributions if you want to. [Learn more about paying extra contributions on the teachers' pension scheme website](https://www.teacherspensions.co.uk/members/working-life/paying-in/increasing-your-pension/additional-pension.aspx).

Reviewer 2: ```suggestion
If your salary increases, the pension contributions you and your employer make will increase too. You can also choose to pay extra contributions if you want to. [Learn more about paying extra contributions on the teachers' pension scheme website](https://www.teacherspensions.co.uk/members/working-life/paying-in/increasing-your-pension/additional-pension.aspx).
```

Appreciate this is for SEO but I think it is slightly clunky - worth rewording?
